docs/guides/formulasC2/examples.md

```diff
@@ -0,0 +1,275 @@
+| :------------- | :------------- |:------------- |:------------- | :------------- |
+|  [Event] (reference.html/#event) |  <ul><li>[JS Script] (reference.html/#js-script)</li><li>[Element API Request] (reference.html/#element-api-request)</li></ul>  | [Element Instance] (reference.html/#element-instance-variable) | <ul><li>CRM hub element instance with events</li><li>Messaging hub authenticated element instance</li></ul> | [Formula JSON][4]  |
+
+To create a formula that listens for an event and emails a message:
+
+1. [Build a formula template](#build-a-formula-template) and select **Event** as the trigger.
+2. Because the trigger is a change to a CRM element, add an element instance variable that refers to a CRM element.
+  3. Click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage">.
+  3. Click **Add New Variable**, and then click **Element Instance**.
+  4. Enter a name for your CRM variable. In this example, we'll use `crmElement`.
+  5. Click **Save**.
+  6. Select the variable that you just created (`crmElement`), and then click **Save** on the Edit event: "trigger" page.
+
+        Your formula visualization should like like the following example:
+        ![Trigger](img/viz-trigger.png)
+
+4. Add another element instance variable for the messaging element.
+  5. Click **Variables**.
+  ![Variables](img/variables.png)
+  7. Click **Element Instance**.
+  8. Enter a name. For this tutorial we'll call it `messagingElement`.
+  9. Click **Save**.
+5. In the formula visualization, click <img src="img/btn-add-step.gif" alt="Add a Step" class="inlineImage"> to add a step.
+6. Create a JS Script step that constructs a message when the trigger happens.
+  7. Click **JS Script**.
+  8. Enter a name for the script. We'll call it `constructBody`.
+  8. Enter a script that constructs a message, such as the example below.
+
+        ```js
+        done( {
+          "subject": "CRM Event Occurred",
+          "to": "receipient@cloud-elements.com",
+          "from": "sender@cloud-elements.com",
+          "message": `${trigger.event.objectType} with ID ${trigger.event.objectId} was ${trigger.event.eventType}`
+        });
+        ```
+  1. Click **Save**.
+1. Create an Element API Request step to send the message that you created in the previous step. Under the **constructBody** step, click the **OnSuccess** button  <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage">.
+    7. Select **Element API Request**.
+    8. Enter a name for the step. We'll call it `sendEmail`.
+    9. In **Element Instance Variable**, click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage">, and then select the **messagingElement** variable that we created earlier.
+    9. In **Method**, select **POST** because the formula will submit a POST request to the messaging hub to an email.
+    10. In **API**, enter the API used to send email messages. In this case, enter `/hubs/messaging/messages`.
```

Comment:
Should we show future examples without the `/hubs/:hubName` since that is no longer necessary when making element instance API calls?